packages/storage/src/storage-browser/locationCredentialsStore/create.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable unused-imports/no-unused-vars */
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+	CredentialsLocation,
+	LocationCredentialsHandler,
+	LocationCredentialsStore,
+} from '../types';
+import { LocationCredentialsProvider } from '../../providers/s3/types/options';
+
+import { createStore, getValue, removeStore } from './registry';
+
+export const createLocationCredentialsStore = (input: {
+	handler: LocationCredentialsHandler;
+}): LocationCredentialsStore => {
+	const storeReference = createStore(input.handler);
```

Comment:
Nit since this isn't actually a reference to the store

```suggestion
	const storeSymbol = createStore(input.handler);
```

Reply:
Makes sense. will address this in the implementation PR.